scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/ManageUsers.java

```diff
@@ -0,0 +1,579 @@
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation; version 3 of
+ * the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307
+ * USA
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ *
+ *  Initial developer(s):               The ProActive Team
+ *                        http://proactive.inria.fr/team_members.htm
+ *  Contributor(s):
+ *
+ * ################################################################
+ * $$PROACTIVE_INITIAL_DEV$$
+ */
+package org.ow2.proactive.scheduler.authentication;
+
+import com.google.common.base.Charsets;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.TreeMultimap;
+import org.apache.commons.cli.*;
+import org.apache.commons.io.IOUtils;
+import org.objectweb.proactive.utils.SecurityManagerConfigurator;
+import org.ow2.proactive.authentication.FileLoginModule;
+import org.ow2.proactive.authentication.crypto.CreateCredentials;
+import org.ow2.proactive.authentication.crypto.Credentials;
+import org.ow2.proactive.authentication.crypto.HybridEncryptionUtil;
+import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
+import org.ow2.proactive.utils.Tools;
+
+import java.io.*;
+import java.security.KeyException;
+import java.security.PublicKey;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.TreeMap;
+
+
+/**
+ * Manage user accounts in login.cfg and group.cfg files
+ *
+ * @author The ProActive Team
+ * @see org.ow2.proactive.authentication.crypto.Credentials
+ */
+public class ManageUsers {
+
+    private static final String newline = System.lineSeparator();
+
+    public static final String CREATE_OPTION = "C";
+    public static final String CREATE_OPTION_NAME = "create";
+    public static final String UPDATE_OPTION = "U";
+    public static final String UPDATE_OPTION_NAME = "update";
+    public static final String DELETE_OPTION = "D";
+    public static final String DELETE_OPTION_NAME = "delete";
+
+    public static final String HELP_OPTION = "h";
+    public static final String HELP_OPTION_NAME = "help";
+    public static final String LOGIN_OPTION = "l";
+    public static final String LOGIN_OPTION_NAME = "login";
+    public static final String PWD_OPTION = "p";
+    public static final String PWD_OPTION_NAME = "password";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS2068)